src/test/java/uk/gov/ons/ctp/response/casesvc/state/StateTransitionManagerUnitTest.java

```diff
@@ -580,4 +580,18 @@ public void givenUnknownEligibilityWhenEqLaunchThenComplete() throws CTPExceptio
     // Then
     assertEquals(CaseGroupStatus.COMPLETE, destinationState);
   }
+
+  @Test
+  public void testCaseGroupTransitionFromCompleteToNotStarted() throws CTPException {
+    // Given
+    CaseGroupStatus complete = CaseGroupStatus.COMPLETE;
+
+    // When
+    CaseGroupStatus destinationState =
+        caseGroupStateMachine.transition(
+            complete, CategoryDTO.CategoryName.OFFLINE_RESPONSE_PROCESSED);
+
+    // Then
+    assertEquals(CaseGroupStatus.COMPLETE, destinationState);
```

Comment:
Is this right? I was expecting not started

Reply:
You're right I've fixed this